moqt-core/src/modules/message_type.rs

```diff
@@ -24,13 +24,13 @@ impl MessageType {
     pub fn is_setup_message(&self) -> bool {
         matches!(self, MessageType::ClientSetup | MessageType::ServerSetup)
     }
-    pub fn is_object(&self) -> bool {
+    pub fn is_object_message(&self) -> bool {
         matches!(
             self,
             MessageType::ObjectWithPayloadLength | MessageType::ObjectWithoutPayloadLength
         )
     }
     pub fn is_control_message(&self) -> bool {
```

Comment:
I think it is better to replace `is_object_message` to `is_object`. I create issue